trieste/acquisition/optimizer.py

```diff
@@ -32,6 +35,22 @@
 """ Type variable bound to :class:`~trieste.space.SearchSpace`. """
 
 
+NUM_SAMPLES_MIN: int = 5000
+"""
+The default minimum number of initial samples for :func:`generate_continuous_optimizer` function in
+:func:`automatic_optimizer_selector`, used for determining the number of initial samples in the
+multi-start acquisition function optimization.
+"""
+
+
+NUM_SAMPLES_DIM: int = 1000
+"""
+The default minimum number of initial samples per dimension of the search space for
+:func:`generate_continuous_optimizer` function in :func:`automatic_optimizer_selector`, used for
+determining the number of initial samples in the multi-start acquisition function optimization.
+"""
+
```

Comment:
Perhaps also use this for the random search optimizer?

Reply:
ok, will do

Reply:
It should be per dim? Why not make it consistent?